pkg/runtime/interfaces/application_configuration.go

```diff
@@ -20,6 +22,12 @@ type DbConfigSection struct {
 	ExtraOptions string `json:"options"`
 	// Whether or not to start the database connection with debug mode enabled.
 	Debug bool `json:"debug"`
+	// Max open connection in postgres
+	MaxOpenConnection     int           `json:"maxOpenConnection"`
```

Comment:
you need to copy these over to the db configuration struct, right? https://github.com/flyteorg/flyteadmin/blob/a4abd4cbb0552a905b95b5d1e58e95baef9f7a7e/pkg/rpc/adminservice/base.go#L83 here and elsewhere

Reply:
@kanterov why do we need to pass these config in extraOption. I used gorm approach https://gorm.io/docs/generic_interface.html